src/indexDB.js

```diff
@@ -0,0 +1,14 @@
+import SyncClient from 'sync-client';
```

Comment:
This file should probably go in `src/tools/`. Let's also give it a more descriptive name like`autosave-client.js`.